Automattic/bugomattic
# Add more information and confetti to the next steps components

#### What Does This PR Add/Change?

This adds some final pieces to the "Next Steps" step!

- Adds the "more information" section which lists a bunch of links related to the product/feature area.
- Adds a lil splash of confetti when you finish all the tasks!

#### Testing Instructions

Manual testing:

- `yarn start`
- Go through whole flow
- See info in "More Information" section. You can change the feature area to see other versions!
- Complete all tasks and see some confetti 🎊 

Unit testing: `yarn test:once`

#### Issues



Related to #  
Closes #
